Display zone and wind settings

dstrawberrygirl opened the pull request:
This adds the UI pieces to the main display to add the zone information and the wind information. I've included a small refactor since I was reusing the dial for both the circular control that sets the wind and the view-only wind control on the screen.

pjanik commented:
Thanks, it looks good, but when I play with it here:
http://wildfire.concord.org/branch/display-zone-settings/index.html
I can't really interact with the model. So, I guess it's something wrong with CSS / layering?

dstrawberrygirl commented:
@pjanik thanks for the feedback! I've adjusted the styling to fix that issue, plus I've added the Time Elapsed styling as per the guidelines so we're much closer to spec. Model dimension details are now hidden by default, but can be displayed with the url parameter `showModelDimensions=true` if required.

pjanik commented:
Thanks! Labels were in front of the terrain dialog, so I just pushed it down a bit. Merging.
